Marlin/language_es.h

```diff
@@ -167,5 +167,47 @@
 #define MSG_DELTA_CALIBRATE_Y               "Calibrar Y"
 #define MSG_DELTA_CALIBRATE_Z               "Calibrar Z"
 #define MSG_DELTA_CALIBRATE_CENTER          "Calibrar Centro"
-
+#define MSG_INFO_MENU                       "Inf. Impresora"
+#define MSG_INFO_PRINTER_MENU               "Inf. Impresora"
+#define MSG_INFO_STATS_MENU                 "Estadisticas Imp."
+#define MSG_INFO_BOARD_MENU                 "Inf. Controlador"
+#define MSG_INFO_THERMISTOR_MENU            "Termistores"
+#define MSG_INFO_EXTRUDERS                  "Extrusores"
+#define MSG_INFO_BAUDRATE                   "Baudios"
+#define MSG_INFO_PROTOCOL                   "Protocolo"
+#define MSG_INFO_PRINT_COUNT                "Conteo de impresion"
+#define MSG_INFO_COMPLETED_PRINTS           "Completadas"
+#define MSG_INFO_PRINT_TIME                 "Tiempo total de imp."
+#define MSG_INFO_PRINT_LONGEST              "Impresion mas larga"
+#define MSG_INFO_PRINT_FILAMENT             "Total de Extrusion"
+#define MSG_INFO_PRINT_COUNT                "Impresiones"
+#define MSG_INFO_COMPLETED_PRINTS           "Completadas"
+#define MSG_INFO_PRINT_TIME                 "Total"
+#define MSG_INFO_PRINT_LONGEST              "Mas larga"
+#define MSG_INFO_PRINT_FILAMENT             "Extrusion"
+#define MSG_INFO_MIN_TEMP                   "Temperatura minima"
+#define MSG_INFO_MAX_TEMP                   "Temperatura maxima"
+#define MSG_INFO_PSU                        "Fuente de poder"
+#define MSG_FILAMENT_CHANGE_HEADER          "Cambiar Filamento"
+#define MSG_FILAMENT_CHANGE_OPTION_HEADER   "Opciones de cambio:"
+#define MSG_FILAMENT_CHANGE_OPTION_EXTRUDE  "Extruir mas"
+#define MSG_FILAMENT_CHANGE_OPTION_RESUME   "Resumir imp."
+#define MSG_FILAMENT_CHANGE_INIT_1          "Esperando para empesar"
```

Comment:
Empesar -> Empezar
On Mon, 15 Aug 2016 at 13:09, Scott Lahteine <notifications@github.com>
wrote:

> In Marlin/language_es.h
> <https://github.com/MarlinFirmware/Marlin/pull/4624#discussion_r74721130>:
>
> > +#define MSG_INFO_PRINT_TIME                 "Tiempo total de imp."
> > +#define MSG_INFO_PRINT_LONGEST              "Impresion mas larga"
> > +#define MSG_INFO_PRINT_FILAMENT             "Total de Extrusion"
> > +#define MSG_INFO_PRINT_COUNT                "Impresiones"
> > +#define MSG_INFO_COMPLETED_PRINTS           "Completadas"
> > +#define MSG_INFO_PRINT_TIME                 "Total"
> > +#define MSG_INFO_PRINT_LONGEST              "Mas larga"
> > +#define MSG_INFO_PRINT_FILAMENT             "Extrusion"
> > +#define MSG_INFO_MIN_TEMP                   "Temperatura minima"
> > +#define MSG_INFO_MAX_TEMP                   "Temperatura maxima"
> > +#define MSG_INFO_PSU                        "Fuente de poder"
> > +#define MSG_FILAMENT_CHANGE_HEADER          "Cambiar Filamento"
> > +#define MSG_FILAMENT_CHANGE_OPTION_HEADER   "Opciones de cambio:"
> > +#define MSG_FILAMENT_CHANGE_OPTION_EXTRUDE  "Extruir mas"
> > +#define MSG_FILAMENT_CHANGE_OPTION_RESUME   "Resumir imp."
> > +#define MSG_FILAMENT_CHANGE_INIT_1          "Esperando para empesar"
>
> Characters past 20 will never be seen, so try to make this string shorter.
> I find that a "ruler" helps:
>
> 12345678901234567|90
> Esperando para empesar
>
> —
> You are receiving this because you are on a team that was mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/MarlinFirmware/Marlin/pull/4624/files/7b7f319ed5cd246fc4880e876a8ebbf222b06328#r74721130>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AGcPb5FIN5pmVklNDaoHhixDkFBGFmRRks5qf_SVgaJpZM4JkGy1>
> .
>